plugins/commands/api/src/main/kotlin/OrtCommand.kt

```diff
@@ -34,7 +34,7 @@ import org.ossreviewtoolkit.utils.ort.ORT_CONFIG_FILENAME
 /**
  * An interface for [CliktCommand]-based ORT commands that come as [Plugin]s.
  */
-abstract class OrtCommand(override val descriptor: PluginDescriptor) : CliktCommand(descriptor.id), Plugin {
+abstract class OrtCommand(override val descriptor: PluginDescriptor) : CliktCommand(), Plugin {
```

Comment:
Did you verify that this does this works for all commands?

Reply:
Indirectly, yes: If it changed anything, then I would have needed to regenerate the shell completion scripts due to changes sub-command names.